src/Traversable.php

```diff
@@ -4,7 +4,6 @@
 
 namespace Marcosh\LamPHPda;
 
-use ArrayIterator;
 use IteratorAggregate;
```

Comment:
also here, I guess we could remove also `IteratorAggregate`

Reply:
Sadly no. When I remove it, Psalm is failing.

Reply:
by "remove" I mean adding a `\` in front...

Reply:
True, I missed that!

Reply:
The `\` was added by PHP CS Fixer, not by me... I just used the default rules.